osu.Game/Online/API/Requests/Responses/SoloScoreInfo.cs

```diff
@@ -45,6 +46,9 @@ public class SoloScoreInfo : IHasOnlineID<long>
 
         [JsonConverter(typeof(StringEnumConverter))]
         [JsonProperty("rank")]
+        // ScoreRank is aligned to make 0 equal D. We still want to serialise this (even when DefaultValueHandling.Ignore is used),
+        // so set the default to an impossible value.
+        [DefaultValue(null)]
```

Comment:
Not sure if you tried it but the internet suggests that
```csharp
[JsonProperty("rank", DefaultValueHandling = DefaultValueHandling.Include)]
```
should do the job as well and be perhaps slightly less esoteric?

Reply:
Amazing, it works. I was searching for something like this but didn't find it.